packages/fxa-auth-server/docs/swagger/shared/descriptions.ts

```diff
@@ -112,8 +145,29 @@ const DESCRIPTIONS = {
   queryKeys:
     'Indicates whether a new `keyFetchToken` is required, default to `false`.',
   payload: 'Opaque payload to be forwarded to the device.',
+  paymentMethodId: 'A unique identifier for the payment method.',
+  paymentProvider:
+    'The third-party that assists in the electronic payments (e.g. PayPal, Stripe).',
+  paymentType:
+    'The type of the payment method (e.g., `credit`, `debit`, `prepaid`, or `unknown`).',
+  paypalPaymentError: 'The payment error from PayPal encountered.',
+  planId:
+    'A unique identifier for the [plan](https://stripe.com/docs/api/plans/object).',
+  planMetadata:
+    'Set of key-value pairs used to store additional information about the plan.',
+  planName: 'The name of the plan.',
   ppidSeed:
     'Seed used in `sub` claim generation of JWT access tokens/ID tokens for clients with [Pseudonymous Pairwise Identifiers (PPID)](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/oauth/pairwise-pseudonymous-identifiers.md) enabled. Used to forcibly rotate the `sub` claim. Must be an integer in the range 0-1024. If not specified, it will default to `0`.',
+  previousProduct:
+    'The value of the previous product that the user had been subscribed to.',
+  priceId:
+    'A unique identifier for the [price](https://stripe.com/docs/api/prices/object).',
+  productId: 'A unique identifier for the product purchased.',
+  productMetadata:
+    'Set of key-value pairs used to store additional information about the product.',
```

Comment:
Perhaps we could link to the Ecosystem docs where we discuss product metadata? Or at least metadata generally.